docs/user/setup.rst

```diff
@@ -3,20 +3,20 @@
 Environment setup
 =================
 
-This is what you will be doing, in a nutshell:
+Scala Native has the following minimum system requirements:
 
-* installation of sbt
-* installation of LLVM and Clang
-* installation of native libraries
+* Java 8
```

Comment:
I wrote in the paragraph before "Scala Native has the following _minimum_ system requirements". Should I still add "or newer" for clarity?